app/models/audio_file.rb

```diff
@@ -37,6 +37,14 @@ def check_self
     false
   end
 
+  def convert_and_move(codec_conversion, out_file_name)
```

Comment:
I think we could improve this naming. `convert_and_move` doesn't clearly distinguish this from `convert`, as they both take the same arguments and the end-result is the same. Maybe something like `convert_with_tmpfile`?